suite-native/module-settings/src/components/AboutViewOnlyBottomSheet.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+    BottomSheet,
+    Box,
+    Button,
+    TitleHeader,
+    VStack,
+    Text,
+    OrderedListItem,
+} from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Translation } from '@suite-native/intl';
+
+const buttonWrapperStyle = prepareNativeStyle(utils => ({
+    gap: utils.spacings.medium,
+}));
+
+type AboutViewOnlyBottomSheetProps = {
+    isVisible: boolean;
+    onClose: () => void;
+};
+
+export const AboutViewOnlyBottomSheet = ({ isVisible, onClose }: AboutViewOnlyBottomSheetProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    let order = 1;
+
+    if (!isVisible) {
+        return null;
+    }
+
+    return (
+        <BottomSheet isVisible={isVisible} onClose={onClose} isCloseDisplayed={false}>
+            <VStack spacing="large" paddingHorizontal="small">
+                <TitleHeader
+                    title={<Translation id="moduleSettings.viewOnly.about.title" />}
+                    subtitle={<Translation id="moduleSettings.viewOnly.about.subtitle" />}
+                />
+                <VStack>
+                    <Text variant="callout">
+                        <Translation id="moduleSettings.viewOnly.about.contentTitle" />
+                    </Text>
+                    <VStack spacing={0}>
+                        <Translation
+                            id="moduleSettings.viewOnly.about.content"
+                            values={{
+                                li: chunk => {
+                                    const index = order;
+                                    order += 1;
+
+                                    return (
+                                        <OrderedListItem
+                                            key={`${chunk}`}
+                                            order={index}
+                                            variant="body"
+                                            color="textSubdued"
+                                        >
+                                            {chunk}
+                                        </OrderedListItem>
+                                    );
+                                },
+                            }}
+                        />
+                    </VStack>
+                </VStack>
+                <Box style={applyStyle(buttonWrapperStyle)}>
```

Comment:
This Box has only one child. Does the `gap` style from `buttonWrapperStyle` have any effect here?